tidalapi/session.py

```diff
@@ -633,12 +633,12 @@ def save_session_to_file(self, session_file: Path):
                 "is_pkce": {"data": self.is_pkce},
                 # "expiry_time": {"data": self.expiry_time},
             }
-            with session_file.open("w") as outfile:
+            with open(session_file, "w") as outfile:
                 json.dump(data, outfile)
 
     def load_session_from_file(self, session_file: Path):
         try:
-            with open(session_file) as f:
+            with open(session_file, "r") as f:
```

Comment:
Adding this is redundant and linters will scream at you for it:

https://docs.python.org/3/library/functions.html#open

Reply:
This change does not make sense, IMO. We should make use of the Path object and call .open() on it.